doc/services/debugging/coredump.rst

```diff
@@ -62,6 +64,21 @@ This usually involves the following steps:
    ``ZEPHYR_TOOLCHAIN_VARIANT=zephyr`` should use the debugger in the
    ``xtensa-intel_apl_adsp`` toolchain of the SDK.
 
+5. When ``DEBUG_COREDUMP_BACKEND_FLASH_PARTITION`` is enabled the core dump
+   data is stored in the flash partition. The flash partition must be defined
+   in the device tree:
+
+	.. code-block:: devicetree
+
+		&flash_sim0 {
+			partitions {
```

Comment:
This code was copied from qemu_x86 overlay https://github.com/zephyrproject-rtos/zephyr/blob/main/tests/subsys/debug/coredump_backends/boards/qemu_x86.overlay :)

Reply:
OK, qemu_x86 only has flash_simulator to simulate flash. It would be better not to refer to this special case, but to use &flash0 instead (as found in your dts files).